torch/testing/_internal/common_utils.py

```diff
@@ -4996,3 +4997,16 @@ def repl_frame(m):
         s = re.sub(r"Cannot export model.+\n\n", "", s)
     s = re.sub(r" +$", "", s, flags=re.M)
     return s
+
+def get_backend_op_dict():
```

Comment:
```suggestion
def get_backend_op_for(device="xpu"):
```

Reply:
I checked TEST_XPU in the function, do we still need device="xpu"?